package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@signalk/charts-plugin",
-  "version": "3.0.0",
+  "version": "3.1.0",
```

Comment:
One the the advantages of "changelog from merged PRs" process is that a PR need not (and should not) change the package version, as there is no guarantee in which version a PRs changes are published.

Reply:
reverted.